src/metrics/metrics.jl

```diff
@@ -29,10 +29,12 @@ const IS_NOT_RELATIVE = false
 struct Metric{F<:Function,T<:Tuple,S<:String,B<:Bool} <: Outcome
     func::F
     dims::T     # output dimension axes ?
+    feature::S
```

Comment:
Clarification question here: What does "feature" relate to?

Reply:
`feature` should be what this metric measure. Coral cover measure "cover" (or "area", if you want to be more precise). That's different from the name/title of a metric. My idea was that when you plot we want to show the plot name ("Absolute Shelter Volume") on top but on the x label (for the series) we want to show what is being measured (volume in m³: `Volume [m³]`).